src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -110,7 +110,7 @@ void deleteParticipantAndAllNeighbors(
      * @param studyId - studyId
      */
     @DELETE( BASE + STUDY_ID_PATH )
-    void deleteStudyAndAllNeighbors(
+    Void deleteStudyAndAllNeighbors(
```

Comment:
So I think the recommended thing to do according to Retrofit is to use a `Response` object.